maintainers/maintainer-list.nix

```
@@ -2798,6 +2798,11 @@
     githubId = 26877687;
     name = "Yurii Izorkin";
   };
+  jacg = {
+    name = "Jacek Generowicz";
+    email = "jacg@my-post-office.net";
+    github = "jacg"
```

```suggestion
    githubId = 2570854;
```
I should have been more specific